test/vending/VendingMachine.test.js

```diff
@@ -45,10 +65,24 @@ describe("VendingMachine", () => {
     await wrappedToken.mint(tokenHolder.address, initialHolderBalance)
   })
 
+  describe("setup", () => {
+    context("once deployed", () => {
+      it("token contract addresses should be correct", async () => {
+        expect(await vendingMachine.tToken()).to.equal(tToken.address)
+        expect(await vendingMachine.wrappedToken()).to.equal(
+          wrappedToken.address
+        )
+      })
+      it("conversion ratio was computed correctly", async () => {
+        expect(await vendingMachine.ratio()).to.equal(expectedRatio)
+      })
+    })
+  })
+
   describe("wrap", () => {
     context("when caller has no wrapped tokens", () => {
       it("should revert", async () => {
-        const amount = 1
+        const amount = to1e18(42)
```

Comment:
Maybe just `to1e18(1)`? It took me a while to think why 42 and I think it's just a random value, right?

Reply:
Oh well, it's simply that 42 is the Answer to the Ultimate Question of Life, the Universe, and Everything, but yeah, `to1e18(1)` works fine too.